src/lib/DepositBootstrap.js

```diff
@@ -30,7 +30,6 @@ class DepositBootstrapComponent extends Component {
       <BaseForm
         onSubmit={this.props.submitFormData}
         formik={{
-          enableReinitialize: true,
```

Comment:
Is there a reason why we reset the form when `initialValues` changes?

(I tried to create/edit/publish records and works OK without it)

Reply:
This is needed, otherwise files are not working properly